### PR TITLE
Feature/POST content endpoint

### PR DESCRIPTION
**Pull request by emilssolmanis**
Adds basic endpoint to ingest `Content` via Deer.

### CODE REVIEW

File: atlas-api/src/main/java/org/atlasapi/util/ClientModelConverter.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            item.setId(codec.decode(api.getId()).longValueExact());
+        }
+
+        // TODO: immutable collector
+        if (api.getAliases() != null) {
+            item.setAliases(api
+                            .getAliases()
+                            .stream()
+                            .map(clientAlias -> new Alias(clientAlias.getNamespace(), clientAlias.getValue()))
+                            .collect(Collectors.toList())
```
Comment (Dalamar42) [inline atlas-api/src/main/java/org/atlasapi/util/ClientModelConverter.java]: atlas-deer should already have ImmutableCollectors somewhere I think

File: atlas-api/src/test/java/org/atlasapi/util/ClientModelConverterTest.java [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+    @Before
+    public void setUp() {
+        converter = new ClientModelConverter();
+    }
+
+    @Test
+    public void convertsContent() {
+        /*
+          Still inconsistent:
+            DisplayTitle displayTitle, -- doesn't really exist on Content, is calculated
```
Comment (Dalamar42) [inline atlas-api/src/test/java/org/atlasapi/util/ClientModelConverterTest.java]: If that field doesn't exist in Content then I think it should be deleted from the client model.

File: atlas-api/src/test/java/org/atlasapi/util/ClientModelConverterTest.java [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+            String priority, -- it's a compound object, not a String
```
Comment (Dalamar42) [inline atlas-api/src/test/java/org/atlasapi/util/ClientModelConverterTest.java]: That has to do with changes Martins is making. Check with him

File: atlas-api/src/test/java/org/atlasapi/util/ClientModelConverterTest.java [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+            List<String> certificates, -- has a country too
```
Comment (Dalamar42) [inline atlas-api/src/test/java/org/atlasapi/util/ClientModelConverterTest.java]: Same for the rest. If you are seeing differences then I think the model should be fixed.

**Comment by Dalamar42** [comment]
Just a couple of comments. Other than that looks good to me.

File: atlas-api/src/test/java/org/atlasapi/util/ClientModelConverterTest.java (hunk 4 above)
Comment (emilssolmanis) [inline atlas-api/src/test/java/org/atlasapi/util/ClientModelConverterTest.java]: @Dalamar42 tl; dr is Tom said to not bother right now as we only need a basic `Content` ingestion thing and should fix as we stumble upon things. I'm generally of the mindset of "do once and forget about it later" though, so I guess we can have a chat about it.

Comment (Dalamar42) [inline atlas-api/src/test/java/org/atlasapi/util/ClientModelConverterTest.java]: @emilssolmanis I tend to be of the same mindset. Let's have a couple min chat tomorrow.

**Pull request closed without merging**